UefiTestingPkg/FunctionalSystemTests/MpManagement/Driver/AARCH64/SuspendHandling.c

```diff
@@ -586,6 +586,9 @@ CpuArchSleep (
     goto Done;
   }
 
+  ArmDisableMmu ();

```

Comment:
I take it on the coming back up path we re-enable the MMU already?

Reply:
yeah, it is part of the existing routine already.